src/main/java/uk/gov/hmcts/sscs/service/CcdService.java

```diff
@@ -107,4 +108,23 @@ public CcdCase findCcdCaseByAppealNumber(String appealNumber) throws CcdExceptio
         return responseEntity.getBody().getCaseData();
     }
 
+    public String unsubscribe(String appealNumber, String reason) throws CcdException {
+
+        String benefitType = null;
+        try {
+            Subscription subscription = new Subscription();
+            subscription.setEmailSubscribe(Boolean.FALSE);
+            subscription.setMobileSubscribe(Boolean.FALSE);
+            subscription.setReason(reason);
```

Comment:
Can this just be passed into a constructor?

Reply:
we can pass it but not sure what other details may come in more as aaron is still working excel doc. For now i will change it to pass through constructor.